### PR TITLE
[DONE] Dirk error handling => snackbar

### PR DESCRIPTION
**Pull request by d-weeteling**

### CODE REVIEW

**Comment by d-weeteling** [comment]
[Errorhandling --> snackbar [1]](https://trello.com/c/PM6wkm9H/1862-errorhandling-snackbar-1)

File: src/actions/ParcelActions.js [hunk 1]
```diff
@@ -1,24 +1,82 @@
...
+});
+
+function handleInvalidDataFormatError(
+  dispatch,
+  placeName,
+  parcelId,
+  parcelGeoserverId
+) {
+  const msg =
+    "Returned geoserver data for parcel with hydracoreID=" +
```
Comment (gijs) [inline src/actions/ParcelActions.js]: Mag dit alsjeblieft met backticks?!??! ;) plz plz plz

File: src/components/views/ListSearchView.jsx [hunk 2]
```diff
@@ -82,10 +71,12 @@ class ListSearchViewComponent extends Component {
 // Local sub-components ///////////////////////////////////////////////////////
 ///////////////////////////////////////////////////////////////////////////////
 
-function ListSearchLanding({ t }) {
+function ListSearchLanding({ t, username }) {
   return (
     <div className={styles.ListSearchLanding} id="ListSearchLanding">
-      <h1 className={styles.Welcome}>{t("Welcome")}</h1>
+      <h1 className={styles.Welcome}>
+        {t("Welcome") + ", " + (username || "Guest")}
```
Comment (gijs) [inline src/components/views/ListSearchView.jsx]: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals

File: src/actions/ParcelActions.js [hunk 3]
```diff
@@ -1,24 +1,82 @@
...
+}
+
+function handleGeoserverError(
+  dispatch,
+  placeName,
+  parcelId,
+  parcelGeoserverId,
+  errorMsg
+) {
+  const msg =
```
Comment (gijs) [inline src/actions/ParcelActions.js]: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals

File: src/actions/SearchActions.js [hunk 4]
```diff
@@ -36,7 +43,18 @@ function doSearch(dispatch, q, types = null, exclude = []) {
   }
 
   dispatch(startSearch());
-  getParcelsByName(q).then(results => dispatch(receiveResults(results)));
+  getParcelsByName(q).then(
+    results => dispatch(receiveResultsSuccess(results)),
+    error => {
+      const msg = "Search error: " + error;
```
Comment (gijs) [inline src/actions/SearchActions.js]: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals

File: src/actions/SearchActions.js [hunk 5]
```diff
@@ -36,7 +43,18 @@ function doSearch(dispatch, q, types = null, exclude = []) {
...
+      dispatch(receiveResultsError(msg));
+      showSnackBar(dispatch, {
+        isError: true,
+        message: "There was an error while searching for '" + q + "'",
```
Comment (gijs) [inline src/actions/SearchActions.js]: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals

File: src/actions/ParcelActions.js [hunk 6]
```diff
@@ -1,24 +1,82 @@
...
+  console.error(msg);
+  dispatch(receiveAttributesFromGeoserverErrorAction(parcelId, errorMsg));
+  showSnackBarParcelReceiveError(dispatch, placeName);
+}
+
+function showSnackBarParcelReceiveError(dispatch, placeName) {
+  const options = {
+    isError: true,
+    message: "There was an error while fetching the parcel details.",
+    subMessage: "No details found for " + placeName
```
Comment (gijs) [inline src/actions/ParcelActions.js]: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals

**Comment by gijs** [comment]
Ziet er goed uit. 1 comment over template literals ipv string concatenation.
I18n doen we later, ik heb de README aangepast met toelichting over hoe dat in bijv. de redux actions kan worden gedaan.

**Pull request merged**